### ISSUE TITLE
Feature Request: Verify User E-Mail Adress in Admin Dashboard

### ISSUE THREAD

**Issue by muellercornelius** [pocketbase/pocketbase#553]
Hi Guys ;),

I thought about migrating one of my apps to pocketbase. And my idea was to keep my old login (strapi) and on successful login i would use the data to create a pocketbase user until all users are migrated. The problem is there are no possibilities to mark the email as verified beside sending the mail to the users. But this would be very confusing. Now I am thinking about a way to handle this. We could expose an new Admin Endpoint? 

Thank you very much for your time and i am looking forward to hear others opinions ;).

Have a nice day ;).

**Comment by ganigeorgiev** [comment]
This is already implemented as part of the ongoing users and profiles refactoring in #376. It will be available in v0.8.0.

---

If your application doesn't use the "verified" state, then you can skip it.
Currently users can authenticate just fine without requiring verification (the `verified` state was intented as identifier whether the email could be trusted and for custom checks in the collections API rules to restrict access to certain areas of the application).

If you want your created users to be marked automatically as "Verified", you can extend PocketBase and use for example the `OnModelBeforeCreate` hook and set it manually (_I haven't tested it_):

```go
// main.go - minimal PocketBase app
// to build run: go build main.go
package main

import (
	"log"

	pb "github.com/pocketbase/pocketbase"
	"github.com/pocketbase/pocketbase/core"
)

func main() {
	app := pb.New()

	app.OnModelBeforeCreate().Add(func(e *core.ModelEvent) error {
		if user, ok := e.Model.(*models.User); ok {
			user.Verified = true
		}
		return nil
	})

	if err := app.Start(); err != nil {
		log.Fatal(err)
	}
}
```

**Comment by muellercornelius** [comment]
wuhuuu. Thank you very much for your quick help. Thats amazing ;).
